components/jupyter-web-app/kubeflow_jupyter/default/templates/notebooks.html

```diff
@@ -1,80 +1,99 @@
 {% extends "layout.html" %}
 {% block javascript %}
-  {{ super() }}
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.min.js"></script>
-  <script src="{{ prefix + url_for('static', filename='js/notebooks.js') }}"></script>
+	{{ super() }}
+    <script defer>
+        // Establish communication with Central Dashboard
+        window.addEventListener('DOMContentLoaded', function (event) {
+            const CD_EvHandler = (window.centraldashboard || {}).CentralDashboardEventHandler;
+            if (!CD_EvHandler) return;
+            CD_EvHandler.init(function(handler, isIframed) {
+                document.body.classList[isIframed?'add':'remove']('iframe-view');
```

Comment:
This feels a bit terse compared to the approach using an `if` block and explicitly calling the `add` or `remove` method on classList. Maybe I'm just slow haha but is there a guide you've consulted in the past as to when it's preferable to use this style over if blocks?

Also, is there some style or trigger being applied to `body.iframe-view`?

Reply:
I've found the js-esque method to be condensed and readable. Simple articles like [this one](https://medium.com/@jraleman/ternary-operators-vs-if-else-statements-6c26f7d034f7) kind of articulate what I mean. A person needs to end up writing a lot more code (therefore a lot more re-reading later) when using if:

```js
document.body.classList[isIframed?'add':'remove']('iframe-view');
```
VS
```js
if (isIframed) {
    document.body.classList.add('iframe-view');
} else {
    document.body.classList.remove('iframe-view');
}
```
I see the former being more readable and doesn't increase code depth. But maybe we can actually come up with our own addendum to the style guide and establish common run-ins such as these?

---

To your secode question, it was, but I removed it in favor of `.disabled`, though maybe it makes sense to scope all SDK behavior under the iframe-view, by prepending all CSS rules with:
```css
body.iframe-view /* normal selector */ {/* css here */}

Reply:
It might be worthwhile to discuss style decisions with the team. Jon has JS readability so he would be a good resource to weigh in on gray areas. I definitely think we should adhere to Google style guide rules as much as possible because the collective wisdom of Google engineering over the years is far better than anything I could come up with.

Reply:
/cc @jharbuck